src/Commands/DrupalCommands.php

```diff
@@ -199,16 +201,16 @@ public function sitePreInstall()
      * @return \Robo\Collection\CollectionBuilder
      */
     public function drushSetup(array $options = [
-      'root' => InputOption::VALUE_REQUIRED,
-      'config-dir' => InputOption::VALUE_REQUIRED,
-    ])
+            'root' => InputOption::VALUE_REQUIRED,
+            'config-dir' => InputOption::VALUE_REQUIRED,
+        ])
     {
         $config = $this->getConfig();
-        $yaml = Yaml::dump($config->get('drupal.drush'));
+        $yaml   = Yaml::dump($config->get('drupal.drush'));
```

Comment:
Let's not align assignment operators, that's completely useless and just causes needless work.